app/build.gradle

```diff
@@ -35,15 +34,21 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    buildFeatures {
+        viewBinding true
+    }
+   namespace 'com.example.life_is_well'
 }
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.core:core-ktx:1.8.0'
     implementation 'androidx.appcompat:appcompat:1.5.0'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation 'androidx.fragment:fragment-ktx:1.5.2'
+    implementation 'androidx.preference:preference:1.1.+'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.2'
```

Comment:
Moving forward we need to discuss adding or updating dependencies before doing them. These affect a large amount of files and memory, so we want to all be an agreement about updating/adding them.